README.md

```diff
@@ -18,9 +18,10 @@ Add your answers to this markdown file.
 Three reasons what asymptotic analysis may be misleading with respect to actual performance in practice are hardware, constant factors, and run times. When an algorithm is ran on two seperate computers, if one computer has better hardware it will run faster. This can be misleading if the two computers are comparing times. This can be the case for two equal computers as well, just with slightly different parts, there is factors like the memory and CPU that could change results. An algorithm can ignore constant factors when using asymptotic analysis. Larger constant factors can make the algorithm slower for practical input sizes. Something else that can be misleading about asymptotic analysis is that two algorithms may have the same asymptotic complexity but have different run times. 
 
 
-Using asymptotic complexity, if a binary search tree with 1,000 elements takes 5 seconds, a binary search tree with 10,000 elements would take roughly 3 times longer. The time complexity of a binary search tree is O(h), where h is the height. The asymptotic complexity of a tree is O(logN). If you 10,000 elements and divide it by the elements for the case that took 5 second, 10,000/1,000 you get 10. If you take log base 2 of 10, you roughly get three, which is the ratio of the two compared. So the time of a list with 1,000 elements takes 5 seconds, and the time that a list of 10,000 elements would tak is, Log base 2 (10)= 3 * 5 seconds = 15 seconds. 
+The asymptotic time complexity of a binary serach tree is O(log n). The best way to find the run time will be to take the ratio of 1,000 elements and 10,000 elements. To do this we can do the following calculation, log2(10,000) = 13.29. We can use the value of 5 seconds for 1,000 elements to figure out the time for 10,000 elements. Do the calculation (13.29 * 5) and we get that the run time will be apporxiamtly 66.54 seconds. (I'm not sure if I am failing to understand this, or if I need to do log2(10,000)/log2(1000) and take that number times 5).
```

Comment:
Why are you multiplying $\log_2 1000$ and $5$? That doesn't seem to have any relationship to the 10000 elements.